src/libAtomVM/bitstring.h

```diff
@@ -90,7 +90,13 @@ extern "C" {
 enum BitstringFlags
 {
     LittleEndianInteger = 0x2,
-    SignedInteger = 0x4
+    SignedInteger = 0x4,
+    NativeEndianInteger = 0x10,
```

Comment:
Did we skip 0x8 for any reason?

Reply:
https://github.com/erlang/otp/blob/48be617b5ad308fcd562a4946e04d6951563e0b4/erts/emulator/beam/erl_bits.h#L214-L218

0x8 is `exact`, this PR only is about `native` :D